src/torchmetrics/functional/audio/pesq.py

```diff
@@ -83,6 +83,8 @@ def perceptual_evaluation_speech_quality(
         )
     import pesq as pesq_backend
 
+    _filter_error_msg = np.vectorize(lambda x: np.issubdtype(type(x), np.number))
```

Comment:
We had problems in the past with `lambda` functions. Could we instead have this as normal function outside the scope of `pesq` function?